code/datums/diseases/advance/symptoms/cough.dm

```diff
@@ -67,7 +67,9 @@
 				else
 					off_cooldown_coughed = FALSE
 			if(affected_mob.CanSpreadAirborneDisease())
-				active_disease.spread(spread_range)
+				if(active_disease.spread_flags != DISEASE_SPREAD_FALTERED)
+					active_disease.spread(spread_range)
+					affected_mob.emote("spin")
```

Comment:
why?

Reply:
spaceacilin, if symptom is faltered it should not spread.

Reply:
but yea im not sure were that spin comes from...silly me.